src/apiCalls.js

```diff
@@ -0,0 +1,17 @@
+const movieData = () => {
+  return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+  .then(response => response.json())
+}
+
+
+// Potential method for dealing with server errors 👇
```

Comment:
We still need to get clarification on how to handle a 500 error code and what we need to be doing.